main.go

```diff
@@ -133,6 +134,69 @@ func main() {
 
 	//Kubeconfig flag
 	flagset.StringVar(&cfg.kubeconfigLocation, "kubeconfig", "", "Path to a kubeconfig file, specifying how to connect to the API server. If unset, in-cluster configuration will be used")
+}
+
+type configfile struct {
+	AuthorizationConfig *authz.Config `json:"authorization,omitempty"`
+}
+
+func (cfg *config) ApplyTo(secureServingInfo **server.SecureServingInfo, loopbackClientConfig **rest.Config, authenticationInfo *server.AuthenticationInfo) error {
```

Comment:
Yes, this is here because of https://github.com/kubernetes/apiserver/blob/edc3b6f28ef9b20877b9b7f9b4cb7fbf004abdc0/pkg/server/options/serving_with_loopback.go#L39.